src/Components/src/Microsoft.AspNetCore.Components.Build/targets/RazorCompilation.targets

```diff
@@ -1,436 +1,36 @@
 <Project>
+  <!-- 
+    All of the core logic for components is now provided by the Razor SDK.
 
-  <!-- Require rebuild if the targets change -->
+    This file now contains some *very temporary* settings to make projects
+    that use components use similar defaults to 2.X-based Blazor projects. 
+  -->
   <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+    <_EnableAllInclusiveRazorSdk>true</_EnableAllInclusiveRazorSdk>
+    <RazorLangVersion>3.0</RazorLangVersion>
+    <RazorDefaultConfiguration>MVC-3.0</RazorDefaultConfiguration>
+    <GenerateRazorAssemblyInfo>false</GenerateRazorAssemblyInfo>
+    <GenerateRazorHostingAssemblyInfo>false</GenerateRazorHostingAssemblyInfo>
+    <GenerateProvideApplicationPartFactoryAttribute>false</GenerateProvideApplicationPartFactoryAttribute>
```

Comment:
We haven't yet defined the MSBuild knobs and dials for a component library, and what set of gestures are on/off by default VS requiring configuration. We don't expect to ship like this 😆